src/main/java/org/tron/core/db/api/index/TransactionIndex.java

```diff
@@ -27,6 +27,7 @@
   public static Attribute<WrappedByteArray, String> OWNERS;
   public static Attribute<WrappedByteArray, String> TOS;
   public static Attribute<WrappedByteArray, Long> TIMESTAMP;
+  public static Attribute<WrappedByteArray, Long> INDEX_CREATE_TIMESTAMP;
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: Make INDEX_CREATE_TIMESTAMP a static final constant or non-public and provide accessors if needed. ([squid:ClassVariableVisibilityCheck](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AClassVariableVisibilityCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1276&issues=AWUdXkliyxJLWg291tgW&open=AWUdXkliyxJLWg291tgW)

[ISSUE_KEY]: AWUdXkliyxJLWg291tgW